src/mlstacks/utils/yaml_utils.py

```diff
@@ -57,9 +58,20 @@ def load_component_yaml(path: str) -> Component:
 
     Returns:
         The component model.
+
+    Raises:
+        FileNotFoundError: If the file is not found.
     """
-    with open(path) as file:
-        component_data = yaml.safe_load(file)
+    try:
+        with open(path) as file:
+            component_data = yaml.safe_load(file)
+    except FileNotFoundError as exc:
+        # Not sure what to do here, as I'd like to specify the path that
+        # caused the problem, but I don't think that's possible while using
+        # a constant for the error message.
+        error_message = f"""Component file at "{path}" specified in 
+                        the stack spec file could not be found."""
+        raise FileNotFoundError(error_message) from exc
```

Comment:
I think you can probably get away with single double quotes here instead of the triple. Did you find this works when you test it locally? @coderabbitai wdyt?

Reply:
Yea, I used triple double quotes for the reasons coderabitai mentioned. I tested it locally, and it seemed to work the way I intended.

Reply:
Would you prefer I change it to single double quotes?

Reply:
Sure, sounds good!

Reply:
Got rid of the comments. Pull request resubmitted.